community/kernel/src/main/java/org/neo4j/kernel/impl/locking/Locks.java

```diff
@@ -78,14 +78,14 @@ void visit( ResourceType resourceType, long resourceId, String description, long
          * Can be grabbed when there are no locks or only share locks on a resource. If the lock cannot be acquired,
          * behavior is specified by the {@link WaitStrategy} for the given {@link ResourceType}.
          */
-        void acquireShared( ResourceType resourceType, long resourceId ) throws AcquireLockTimeoutException;
+        void acquireShared( ResourceType resourceType, long... resourceIds ) throws AcquireLockTimeoutException;
```

Comment:
We need to add into java doc that those resources ids should be ordered in some way